examples/samv71-hal-watchdog/src/main.rs

```diff
@@ -0,0 +1,102 @@
+#![no_std]
+#![no_main]
+
+extern crate cortex_m;
+extern crate cortex_m_rt as rt;
+extern crate panic_semihosting;
+
+use aerugo::hal::peripherals::watchdog::{Watchdog, WatchdogConfig};
+use aerugo::{
+    InitApi, MessageQueueHandle, MessageQueueStorage, TaskletConfig, TaskletStorage, AERUGO,
+};
+use cortex_m_semihosting::hprintln;
+use rt::entry;
+
+struct WdtTaskContext {
+    watchdog: Watchdog,
+    counter: u32,
+    queue_handle: MessageQueueHandle<u32, 10>,
+}
+
+fn watchdog_task(_counter: u32, context: &mut WdtTaskContext) {
+    context.watchdog.feed();
+    context.counter = context.counter.wrapping_add(1);
+
+    context
+        .queue_handle
+        .send_data(context.counter)
+        .expect("Unable to send data from task.");
+}
+
+static WATCHDOG_TASK_STORAGE: TaskletStorage<u32, WdtTaskContext> = TaskletStorage::new();
+static DATA_QUEUE: MessageQueueStorage<u32, 10> = MessageQueueStorage::new();
```

Comment:
Rewrite this example to use cyclic tasklet

Reply:
I'll just kill this example, since i'm planning to completely rewrite it in separate MR.

Reply:
ack